contributors/design-proposals/hugepages.md

```diff
@@ -125,7 +166,41 @@ pages, and Quota should be able to count them.
 
 Get design approval
 
-### Phase 1: Add huge page support
+### Phase 1: Huge page volume plugin
+
+**Target 1.8+**
+
+Implement huge page volume plugin. Accounting and limiting huge pages before 
+`Phase 2` can be done via OIR(opaque integer resources). 
+
+pkg/api/types.go ( and v1/types.go)
+
+```
+type VolumeSource struct { 
+...
+    // HugePages represensts a hugepage resource.                      
+    // +optional
+    HugePages *HugePagesVolumeSource `json:"hugePages,omitempty" protobuf:"bytes,28,opt,name=hugePages"` 
+}
+
+// HugePagesSource represents Linux HugeTlbPage https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt                                                                                    
+type HugePagesVolumeSource struct {
+    // Defaults to 2M
```

Comment:
That is architecture specific - defaulting to /proc/meminfo Hugepagesize would allow the proposal to support different archs in the future.